padding/padding_test.go

```diff
@@ -140,6 +151,65 @@ func TestWriter_pad(t *testing.T) {
 	if actual != expected {
 		t.Errorf("expected:\n\n`%s`\n\nActual Output:\n\n`%s`", expected, actual)
 	}
+
+	f.closed = true
+
+	if err := f.pad(); err != ErrClosed {
+		t.Error(err)
+	}
+}
+
+func TestWriter_Flush(t *testing.T) {
+	t.Parallel()
+
+	f := NewWriter(6, nil)
+
+	_, err := f.Write([]byte("foo"))
+	if err != nil {
+		t.Error(err)
+	}
+
+	if err := f.Flush(); err != nil {
+		t.Error(err)
+	}
+
+	exp := "foo   "
+	if f.String() != exp {
+		t.Errorf("expected:\n\n`%s`\n\nActual Output:\n\n`%s`", exp, f.String())
+	}
+
+	_, err = f.Write([]byte("bar"))
+	if err != nil {
+		t.Error(err)
+	}
+	if err := f.Flush(); err != nil {
+		t.Error(err)
+	}
+
+	exp = "bar   "
+	if f.String() != exp {
+		t.Errorf("expected:\n\n`%s`\n\nActual Output:\n\n`%s`", exp, f.String())
+	}
+}
+
+func TestWriter_Close(t *testing.T) {
+	t.Parallel()
+
+	f := &Writer{
+		Padding:    6,
+		lineLen:    1,
+		ansiWriter: &ansi.Writer{Forward: fakeWriter{}},
+	}
+
+	if err := f.Close(); err != fakeErr {
+		t.Error(err)
+	}
+
+	f.closed = true
```

Comment:
Why do you manually set this? Shouldn't `closed` already be true after `f.Close()`?

Reply:
No, `closed` won't be set to true if error occurs.